### PR TITLE
OCPBUGS-29603: OpenStack: document know issue of IPv6 Lb with healthmonitors

### PR DESCRIPTION
**Pull request by MaysaMacedo**
document https://issues.redhat.com/browse/OCPBUGS-29603

Version(s):

Issue:

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by JoeAldinger — COMMENTED** [review]
small change

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -2376,6 +2376,8 @@ In the following tables, features are marked with the following statuses:
 
 * In {product-version} with IPv6 networking running on {rh-openstack-first} environments, `IngressController` objects configured with the `endpointPublishingStrategy.type=LoadBalancerService` YAML attribute will not function correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2263550[*BZ#2263550*], link:https://bugzilla.redhat.com/show_bug.cgi?id=2263552[*BZ#2263552*])
 
+* In {product-version} with IPv6 networking running on {rh-openstack-first} environments, health monitors created with IPv6 OVN-Octavia load balancers will not function correctly. (link:https://issues.redhat.com/browse/OCPBUGS-29603[*OCPBUGS-29603*])
```
[the remark this answers is not in the document]
Comment (MaysaMacedo) [reply release_notes/ocp-4-15-release-notes.adoc]: done

**Comment by JoeAldinger** [comment]
merged asked for in slack

**Pull request merged**